packages/app/ui/modal/PaymentConfirmationModal.tsx

```diff
@@ -122,6 +131,19 @@ export function PaymentConfirmationModal({
             </Text>
             <Text className="my-2 text-sm">Network fee: 0.01 XLM</Text>
             <Line />
+            <GestureHandlerRootView
+              style={{ flexDirection: "row", width: "100%" }}
+            >
+              <SkyhitzSlider
+                minimumValue={1}
+                maximumValue={initialEquityForSale * 100}
+                value={initialEquityForSale * 100}
+                key={entry.id}
+                onValueChange={(value: number) => {
+                  setEquityForSale(+(value / 100).toFixed(2));
```

Comment:
```suggestion
                  setEquityForSale((value / 100).toFixed(2));
```

Reply:
I think we should leave it since toFixed converts number to string.

Reply:
can we use `parseFloat` then?